src/templates/open-enterprise/index.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react'
+import BN from 'bn.js'
+import {
+  ClaimDomainScreen,
+  DotVotingScreen,
+  KnownAppBadge,
+  ReviewScreen,
+  TokensScreen,
+  VotingScreen,
+} from '../kit'
+
+import header from './header.svg'
+import icon from './icon.svg'
+
+function completeDomain(domain) {
+  return domain ? `${domain}.aragonid.eth` : ''
+}
+
+export default {
+  id: 'open-enterprise-template.aragonpm.eth',
+  name: 'Open Enterprise',
+  header,
```

Comment:
Let's add the [`new` and `disabled`](https://github.com/aragon/aragon/blob/master/src/templates/fundraising/index.js#L150-L151) flags like we have in fundraising :).

Reply:
Hardcoding the `disabled` option seems a bit rigid, it will require new PR each time we deploy to each network, I think just with the `id` being found in the APM should be enough to check if the template exists on that network (as it was working before adding the disabled flag)

Just my opinion, anyway I added both flags as they are in the fundraising template.

Reply:
Hmm yes, definitely, maybe we should do this off of chain state instead. We already disable the button in the "template details" modal if we can't resolve the ENS name.

Reply:
But off-chain check does not make lot of sense for me when the goal is to create an on-chain template instance, I am probably missing something here

Reply:
> off of chain state instead

I was meaning to base it on on-chain state (in this case, ENS), so we mean the same.

> the goal is to create an on-chain template instance

This is not necessarily the goal; we need all the pieces to work, but may want to decorate it differently; in this case, we want to disable it even if it's available. We can rethink this later though; perhaps we only want to annotate that it's "coming soon" and place it at the front of the list.